src/uct/ib/base/ib_md.c

```diff
@@ -1524,10 +1498,13 @@ uct_component_t uct_ib_component = {
     .md_vfs_init        = (uct_component_md_vfs_init_func_t)ucs_empty_function
 };
 
+int uct_ib_mlx5_loaded = 0; /* Populated by IB MLX5 submodule */
```

Comment:
minor: why not declaring in the beggining of the file?

Reply:
fixed